app/Cache/CacheServiceProvider.php

```diff
@@ -38,7 +38,7 @@ public function register(): void
         $container->addShared(
             Client::class,
```

Comment:
This approach is not future-proof. If we change the `cache` in the future we need `temporary_cache` and `persistent_cache` as exported services

Reply:
Solved by using `app_cache` and `data_cache`